src/cow-react/modules/limitOrders/state/limitOrdersReceiptAtom.ts

```diff
@@ -0,0 +1,18 @@
+import { atom } from 'jotai'
+import { ParsedOrder } from '@cow/modules/limitOrders/containers/OrdersWidget/hooks/useLimitOrdersList'
+
+export interface ReceiptState {
+  selected: ParsedOrder | null
+}
+
+export const receiptAtom = atom<ReceiptState>({
```

Comment:
Would be great to replace it by `Router` usage (`?orderId=xxxx`)

Reply:
Added todo